working-groups/wg-talent-plan-courses.md

```diff
@@ -2,15 +2,15 @@
 
 ## Goals
 
-The Talent Plan Course Working Group aims to develop a series of courses to help people enter the field of computer infrastructure system and TiDB ecosystem.
+Talent Plan Course Working Group aims to develop a new version of Talent Plan Courses (see [Proposal: Talent Plan Courses](https://github.com/JinLingChristopher/community/blob/fe6991c5bd4a63254e3d116410702d1c59055abb/rfc/2019-12-11-Talent-Plan-Courses.md)) based on Talent Plan 1.0 to help open source and infrastructure enthusiasts have better command of programming language, open source collaboration, infrastructure systems and TiDB ecosystem.
```

Comment:
https://github.com/pingcap/community/blob/master/rfc/2019-12-11-Talent-Plan-Courses.md

this is preferred.

Reply:
thanks a lot